### ISSUE TITLE
Long Running Time Without Output When Using Rosetta for VSH Mode Docking

### ISSUE THREAD

**Issue by YangPH0624** [gfzhou/OpenVS#6]
I am encountering an issue while attempting to replicate the VSH mode docking using Rosetta in OpenVS. Below are the details of my environment and parameters:
Processor: Intel Core i7-12700
Cores Used: 12 cores in parallel
Ligand Count: Only using 1 ligand
Runtime: It has been running for over an hour without any output

I have attempted multiple runs, and the results are consistently the same. Could you provide any possible solutions or adjustments that might help?Thank you for your assistance!
![6ff031b21dbcaf95aea1581af2c36dd](https://github.com/user-attachments/assets/5a8cae7f-3eb5-4289-98c6-fa968c7ad790)
![a3421d83e822847f64316e62f5f2e80](https://github.com/user-attachments/assets/cdb011b9-7e29-4d27-aa63-c16992fc917f)

**Comment by gfzhou** [comment]
Please make sure that the ligand molecule is the last "residue" in your input `complex.pdb` file. e.g.
```
ATOM   6579 N    GLY A 440      -2.833  35.357 -13.985  1.00  0.00           N
ATOM   6580 CA   GLY A 440      -2.669  35.130 -15.410  1.00  0.00           C
ATOM   6581 C    GLY A 440      -2.619  33.643 -15.702  1.00  0.00           C
ATOM   6582 O    GLY A 440      -2.055  33.204 -16.705  1.00  0.00           O
ATOM   6583 H    GLY A 440      -2.774  34.579 -13.342  1.00  0.00           H
ATOM   6584 1HA  GLY A 440      -3.497  35.590 -15.952  1.00  0.00           H
ATOM   6585 2HA  GLY A 440      -1.752  35.615 -15.749  1.00  0.00           H
HETATM 6586 C1   LG1 X 900      -0.030  32.799  22.955  1.00  0.00           C
HETATM 6587 C2   LG1 X 900       0.173  33.973  22.012  1.00  0.00           C
HETATM 6588 C3   LG1 X 900       1.477  34.336  21.541  1.00  0.00           C
HETATM 6589 C4   LG1 X 900       1.667  35.425  20.642  1.00  0.00           C
HETATM 6590 C5   LG1 X 900       0.562  36.219  20.157  1.00  0.00           C
HETATM 6591 C6   LG1 X 900      -0.786  35.858  20.634  1.00  0.00           C
```

**Comment by YangPH0624** [comment]
Thank you very much for your reply. I successfully completed the docking of a single molecule. If I need to perform large-scale molecular docking, do I need to input the structure of each small molecule and the protein complex for the -s option?

**Comment by OYH-2024** [comment]
> 我在尝试在 OpenVS 中使用 Rosetta 复制 VSH 模式停靠时遇到问题。以下是我的环境和参数的详细信息： 处理器：Intel Core i7-12700 核心 使用：12 个核心并行 配体计数：仅使用 1 个配体 运行时间：它已经运行了一个多小时，没有任何输出
> 
> 我尝试了多次运行，结果始终相同。您能否提供任何可能的解决方案或调整？感谢您的帮助！ ![6ff031b21dbcaf95aea1581af2c36dd](https://private-user-images.githubusercontent.com/101262191/378061067-5a8cae7f-3eb5-4289-98c6-fa968c7ad790.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1Mjg4OTAsIm5iZiI6MTczMjUyODU5MCwicGF0aCI6Ii8xMDEyNjIxOTEvMzc4MDYxMDY3LTVhOGNhZTdmLTNlYjUtNDI4OS05OGM2LWZhOTY4YzdhZDc5MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyNVQwOTU2MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYzRkNWU0ODMwNjAyZWMzMDFlYzg0YWM3ODRmYTE1MTk4OThjM2RkNmM1YmE0MjZiMGFmMmE0NjQ5Zjg5NThiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gbwRSqDXITNhmOxphyUiCEuRlW2F1MflF43FKx6Jy6g) ![a3421d83e822847f64316e62f5f2e80](https://private-user-images.githubusercontent.com/101262191/378061075-cdb011b9-7e29-4d27-aa63-c16992fc917f.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1Mjg4OTAsIm5iZiI6MTczMjUyODU5MCwicGF0aCI6Ii8xMDEyNjIxOTEvMzc4MDYxMDc1LWNkYjAxMWI5LTdlMjktNGQyNy1hYTYzLWMxNjk5MmZjOTE3Zi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyNVQwOTU2MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZTkxYjZhYmIzNTYyMDViZDExNjgyNDRlOTQ3ZDk4YTNhOWFmMzU4YTliNzdmMzg2NGUxYzVhNzUzYTBlOTYyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.pH4eUarmoxP3bhO0ndvJ4sbvgpS3PPiscShFUS-qMmU)

I've been parked in this spot for a long time and would love to ask for your advice

**Comment by OYH-2024** [comment]
[FILE]: src/core/chemical/residue_io.cc
[LINE]: 143
[START_MESSAGE]
[ ERROR ] UtilityExitException
ERROR: Cannot find file '../target/KLHDC2_c8_lig.params or /mnt/e/rosetta/rosetta.binary.ubuntu.release-371/main/database/chemical/residue_type_sets/fa_standard/residue_types/../target/KLHDC2_c8_lig.params'   May I ask how this problem can be solved?
